test/jdk/java/net/vthread/HttpALot.java

```diff
@@ -23,13 +23,20 @@
 
 /**
  * @test
+ * @bug 8294610
  * @summary Stress test the HTTP protocol handler and HTTP server
  * @requires vm.debug != true
  * @modules jdk.httpserver
  * @library /test/lib
  * @compile --enable-preview -source ${jdk.version} HttpALot.java
+ * @comment The test launch command intentionally uses -Dsun.net.httpserver.nodelay=true
+ *          to enable TCP_NODELAY on the sockets "accept()"ed by the HttpServer. This is
+ *          to avoid (occasional 40 milli seconds) delays in receiving responses from the server.
+ *          Given the number of requests being fired in this test, the delayed responses can
+ *          accumulate into longer test run duration
  * @run main/othervm/timeout=600
  *     --enable-preview
+ *     -Dsun.net.httpserver.nodelay=true
```

Comment:
good, I'm happy this issue was found and the HTTP server already has a way to change this option.